config/rubocop.yml

```diff
@@ -107,10 +107,31 @@ Style/StringMethods:
 Rails/SaveBang:
   Enabled: true
 
+Rails/SkipsModelValidations:
+  Blacklist:
+    # - update_all
+    - decrement!
+    - decrement_counter
+    - increment!
+    - increment_counter
+    - toggle!
+    - touch
+    - update_attribute
+    - update_column
+    - update_columns
+    - update_counters
```

Comment:
This is the default setting with `update_all` commented out(i.e. it is permitted) as I often find it useful for performance.